### ISSUE TITLE
Collada file iffy root transformation

### ISSUE THREAD

**Issue by rakocevicjovan** [assimp/assimp#3257]
Hello, I'm an assimp user of a few years. I used assimp to load skeletal animations for some time (mixamo models work great) but I ran into an issue with a collada file, which I can attach if you want to take a look at it.

Using open3mod, the root of the scene is as follows:
0 translation in xyz, unifrom 1. scaling, rotation quaternion: -0.707, 0., 0., 0.707.

In my engine, using directX ( I am using the convert to left handed flag, that might be important not sure), the root of the scene is rotated by quaternion: 1 0 0 0, also no translation or scaling.

Now, I am not sure what exactly the root of the scene represents, and this disparity might be expected, but it bugs me because this model also appears distorted and I've been hunting for the reason why (again, my skinning seems to be all right because the mixamo models work, even when they contain multiple meshes and even attached weapons).

There is no concatenation going on, the data describes local transforms of the root node (called simply Scene) so it can't be order of multiplications on my side, I read this root node transform straight from assimp. Conversion to quaternion is there because open3mod uses it, and is correct, in case you were wondering if that's an issue, I verified it, and the matrices do not match either.

I think open3mod is using an older version of assimp, but it displays the mesh properly so I am assuming there's an error in the newer version.

I found a few other models that don't work in a similar fashion, and the only thing I could find connecting them is that they are made in blender. 

![image](https://user-images.githubusercontent.com/22936067/83629854-222b5300-a59b-11ea-8ab0-5299e3163fbd.png)

![image](https://user-images.githubusercontent.com/22936067/83629890-31120580-a59b-11ea-9e97-dac974df6e09.png)

Edit: To explain my confusion about root node transforms, collada file basically doesn't have one. It has one for other nodes. I assume this root transformation is used by assimp to account for coordinate system differences between programs, so this might be expected to be different between open3mod (which I think uses opengl) and my program.

**Comment by rakocevicjovan** [comment]
To add more information, I used the official fbx converter, converted from collada to fbx and tried in UE4, open3mod and my own renderer and it shows the very same (faulty) animation in all three! However fbx review plays it well when converted, and doesn't even detect any animations when it's in collada format... 

I really don't understand what kind of magic is at hand here considering this was the case with several "bad" files where they will be broken in several renderers/engines, even official and what you would expect to be robust ones, but works in one of them somehow... I feel there's a lot of bad exporters out there contributing to this, and a number of incompatible standards imposed by them.

TLDR: This file is probably faulty and it's not a problem in assimp. Feel free to close it if you think so, I gave up on this asset already.

**Comment by rakocevicjovan** [comment]
Confirmed issue is not in assimp, one file was faulty, issue of another was caused by my bug. Closing, cheers.
